components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/util/OIDCErrorConstants.java

```diff
@@ -23,6 +23,9 @@
  */
 public class OIDCErrorConstants {
 
+    public static final String LOGOUT_SERVER_EXCEPTION = "Back channel logout failed due to server error.";
```

Comment:
why do we have separate errormsg here? Can't we have it inside the ErrorMessages enum ad have a generic errormsg there?